loops_lab.js

```diff
@@ -0,0 +1,118 @@
+// // Loops Lab
+// //
+// // Problem One
+// // a.Write a while loop that takes the variable num and logs all the numbers, descending, between "num"
+// // and 1
+
+let numA = 5;
+while (numA > 1) {
+    console.log(numA)
+    numA--;
+}
+console.log(numA)
+
+// // b.Write a
+// // for loop that takes the variable num and logs all the numbers, descending, between "num"
+// // and 1
+
+// // Example:
+
+// //     let num = 8
+
+// // // Your code here
+
+// // /* Logs:
+// // 8
+// // 7
+// // 6
+// // 5
+// // 4
+// // 3
+// // 2
+// // 1
+// // */
```

Comment:
standard is to use i in for loops. It's asking you to start at numB. That way you can use it for different sized loops.